src/sentry/grouping/strategies/configurations.py

```diff
@@ -2,16 +2,17 @@
     RISK_LEVEL_HIGH,
     RISK_LEVEL_LOW,
     RISK_LEVEL_MEDIUM,
+    StrategyConfiguration,
     create_strategy_configuration,
 )
 
 # The full mapping of all known configurations.
-CONFIGURATIONS = {}
+CONFIGURATIONS: dict[str, type[StrategyConfiguration]] = {}
 
 # The implied base strategy *every* strategy inherits from if no
 # base is defined.
 BASE_STRATEGY = create_strategy_configuration(
-    None,
+    "base",
```

Comment:
Interesting. What's the difference?

Reply:
Its mostly just to satisfy the types, as the `id` must not be `None`.

I believe the only visible change this makes is here:
https://github.com/getsentry/sentry/blob/29dddb9250f04ca20d1613f1f776216d2f79dee8/src/sentry/grouping/strategies/base.py#L316

Alternatively, we can also type this as `id: str | None`

Reply:
If you can confirm that it does not change behaviour go ahead with this change.

Pick whatever you think works best.

Reply:
I actually fixed this in a different way in https://github.com/getsentry/sentry/pull/63666